docs/file-reference.md

```diff
@@ -0,0 +1,151 @@
+# OpenCompose file reference documentation
+
+
+OpenCompose file example
+```yml
+version: "0.1-dev"
+
+services:
+   name: foobar
+   containers:
+   - image: foo/bar:tag
+     env:
+     - foo=bar
+     command: ["/bin/foobar"]
+     args: 
+     - "-f"
+     - "/tmp/foobar"
+     ports:
+     - port: 8080:80
+
+```
+
+
+OpenCompose file has two main sections version and services.
+
+## version
+OpenCompose format version
+
+## services
+`services` is main section that lists all the services that this OpenCompose files describes.
+`services` has to be array of [ServiceSpec](#servicespec)
+
+
+### ServiceSpec
+This describes one service. One service doesn't necessarily mean one container.
+One service can be comprised of multiple containers.
+Each service has name and list of the containers.
+
+```yml
+# <ServiceSpec>
+  name: foo
+  containers:
+  - <ContainerSpec>
+  - <ContainerSpec>
```

Comment:
I think one item is enough since this is an array, but I don't have strong opinion on this.